server/enhancedDate.js

```diff
@@ -1 +1,59 @@
-// export this as a Node module using the enhancedDate.js code from Module_2
\ No newline at end of file
+
+var dateIsSet;
+var dayOfWeekArray = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+var monthArray = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+//default function
+function isDateSet() {
+	if(!dateIsSet) {console.warn('date hasn\'t been set! Please call setDate first!'); return false;}
```

Comment:
This function would have more utility if you had it always return a value - currently it will only return false or undefined, when it would be better to return true if dateIsSet evaluates to true.